### PR TITLE
Fix miscalculated Debye-Waller factor in ComputeIncoherentDos.py

### LINKED ISSUE #36306: Misscalculated Debye-Waller factor in ComputeIncoherentDos.py
**Describe the bug**
I noticed that the Debye-Waller factor in the algorithm ComputeIncoherentDos.py is calculated as:
` DWF = np.exp(-2*(qqgridsq*msd))` with `msd` the averaged mean square displacement provided by the user. 

Given the definition from the in the reference page (https://docs.mantidproject.org/nightly/algorithms/ComputeIncoherentDOS-v1.html#algm-computeincoherentdos), the DWF should be:
` DWF = np.exp(-(qqgridsq*msd))`

I think that the factor 2 should not be here.

### PR DESCRIPTION
**Pull request by jhaigh0**
### Description of work

As pointed out by @fucaline the Debye-Waller factor calculation in ComputeIncoherentDos.py was inconsistent with what was stated in the docs. I asked @mducle for his opinion since he implement the algorithm and he helped me with the fix.

Fixes #36306

### To test:

- Try the example usage from the [doc page](https://docs.mantidproject.org/nightly/algorithms/ComputeIncoherentDOS-v1.html#algm-computeincoherentdos)
- Build the algorithm docs and check what I've added displays correctly and makes sense

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: docs/source/algorithms/ComputeIncoherentDOS-v1.rst [hunk 1]
```diff
@@ -28,6 +28,12 @@ The average Debye-Waller factor :math:`\exp\left(-2\bar{W}(Q)\right)` is
 calculated using an average mean-square displacement :math:`\langle u^2 \rangle`,
 using :math:`W=Q^2\langle u^2\rangle/2`.
 
+:math:`\langle u^2 \rangle` is also called the isotropic atomic displacement
+parameter ``Uiso`` :math:`U_{\mathrm{iso}}` in Rietveld refinement programs
```
Comment (jclarkeSTFC) [inline docs/source/algorithms/ComputeIncoherentDOS-v1.rst:32]:
```suggestion
parameter :math:`U_{\mathrm{iso}}` in Rietveld refinement programs
```
I think the formatted U_iso is clear on its own

File: docs/source/algorithms/ComputeIncoherentDOS-v1.rst [hunk 2]
```diff
@@ -28,6 +28,12 @@ The average Debye-Waller factor :math:`\exp\left(-2\bar{W}(Q)\right)` is
...
+like `GSAS <https://subversion.xray.aps.anl.gov/trac/EXPGUI/wiki/GSASIntro>`_.
+There is also another, related, type of atomic displacement parameter used in programs like
+`FullProf <https://www.ill.eu/sites/fullprof/>`_ called ``Biso``: :math:`B_{\mathrm{iso}}=8\pi^2U_{\mathrm{iso}}` [#GSASREF]_.
```
Comment (jclarkeSTFC) [inline docs/source/algorithms/ComputeIncoherentDOS-v1.rst:35]:
```suggestion
`FullProf <https://www.ill.eu/sites/fullprof/>`_ called :math:`B_{\mathrm{iso}}` : :math:`B_{\mathrm{iso}}=8\pi^2U_{\mathrm{iso}}` [#GSASREF]_.
```

Formatted B_iso looks a bit nicer

**Review by jclarkeSTFC — COMMENTED** [review]
Looks fine to me, docs look okay, I tried the three examples from the docs and their output is very close to or the same as in 6.8. I wasn't sure if it should be exactly the same because of the change in the calculation. 

Does it look okay to you @mducle?

One other thing I noticed, unrelated to these changes, but I couldn't find the `mari_res2013.map` file anywhere, I just used the `mari_res.map` file in `SampleData-ISIS`.

**Review by mducle — APPROVED** [review]
Looks good to me.

**Comment by mducle** [comment]
> One other thing I noticed, unrelated to these changes, but I couldn't find the `mari_res2013.map` file anywhere, I just used the `mari_res.map` file in `SampleData-ISIS`.

Ah, it's in the [script repository](https://github.com/mantidproject/scriptrepository/tree/master/direct_inelastic/MARI/MapFiles). I think that `mari_res.map` and `mari_res2013.map` are identical or almost identical so it doesn't matter which is used.

**Pull request merged**